Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=11d518e07d700eeb5bcec36bfd5f501e405230dd
Commit:     11d518e07d700eeb5bcec36bfd5f501e405230dd
Parent:     12dde4c6d6e9092cf63094e84dc1fe4e505dd6d0
Author:     Takashi Iwai <tiwai@suse.de>
AuthorDate: Wed Dec 10 10:37:33 2008 +0100
Committer:  Takashi Iwai <tiwai@suse.de>
CommitDate: Wed Dec 10 10:39:17 2008 +0100

    ALSA: hda - Add quirk for HP6730B laptop
    
    Added model=laptop for HP 6730B laptop with AD1984A codec.
    Reference: Novell bnc#457909
    	https://bugzilla.novell.com/show_bug.cgi?id=457909
    
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Cc: stable@kernel.org

 sound/pci/hda/patch_analog.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/sound/pci/hda/patch_analog.c b/sound/pci/hda/patch_analog.c
index c1918a1..26247cf 100644
--- a/sound/pci/hda/patch_analog.c
+++ b/sound/pci/hda/patch_analog.c
@@ -3901,6 +3901,7 @@ static const char *ad1884a_models[AD1884A_MODELS] = {
 static struct snd_pci_quirk ad1884a_cfg_tbl[] = {
 	SND_PCI_QUIRK(0x103c, 0x3030, "HP", AD1884A_MOBILE),
 	SND_PCI_QUIRK(0x103c, 0x3056, "HP", AD1884A_MOBILE),
+	SND_PCI_QUIRK(0x103c, 0x30e6, "HP 6730b", AD1884A_LAPTOP),
 	SND_PCI_QUIRK(0x103c, 0x30e7, "HP EliteBook 8530p", AD1884A_LAPTOP),
 	SND_PCI_QUIRK(0x103c, 0x3614, "HP 6730s", AD1884A_LAPTOP),
 	SND_PCI_QUIRK(0x17aa, 0x20ac, "Thinkpad X300", AD1884A_THINKPAD),
